Src/Screens/LoggedInScreen/LoggedIn.swift

```diff
@@ -56,13 +56,13 @@ extension LoggedIn: Presentable {
             profilePresentation
         )
         
-        let appVersion = Bundle.main.infoDictionary!["CFBundleShortVersionString"] as? String ?? "0.0.0"
-        let lastNewsSeen = UserDefaults.standard.string(forKey: WhatsNew.lastNewsSeenKey) ?? "0.0.0"
+        let appVersion = Bundle.appVersion()
```

Comment:
This should be `Bundle.main.appVersion()` or?

Reply:
Yeah but now when I changed it into a `var`: `Bundle.main.appVersion`